tools/importer/rules/accordion.js

```diff
@@ -0,0 +1,35 @@
+import { addRow } from '../utils.js';
+
+export default function createAccordionBlocks(main, document) {
+  const accordions = document.querySelectorAll('.accordion');
+
+  // fast return
+  if (!accordions) {
+    return;
+  }
+
+  accordions.forEach((accordion) => {
+    const parentClasses = accordion.parentElement.classList;
+    const items = accordion.querySelectorAll('.spectrum-Accordion-item');
+    const accBlockTable = document.createElement('table');
```

Comment:
I have seen this way of creating the tables in different places, I am not where it is coming from. Check here: https://github.com/kptdobe/helix-imports/blob/main/tools/importer/import-business-acom.js#L104

There is an helper method: `const table = WebImporter.DOMUtils.createTable(cells, document);` where cells is an array of arrays. I find it much easier to handle and to read. But I am biased because I have created the helper ;-)

Reply:
Done, updated in every place 👍 .